src/main/java/com/example/S7/StudentRepository.java

```diff
@@ -1,27 +1,16 @@
 package com.example.S7;
 
 import java.util.List;
-import org.apache.ibatis.annotations.Delete;
-import org.apache.ibatis.annotations.Insert;
 import org.apache.ibatis.annotations.Mapper;
 import org.apache.ibatis.annotations.Select;
-import org.apache.ibatis.annotations.Update;
 
 @Mapper
 public interface StudentRepository {
 
-  @Select("SELECT * FROM student")
-  List<Student> getAllStudents();
+  @Select("SELECT * FROM students")
+  List<Student> Allstudents();
```

Comment:
キャメルケースに沿っていませんね。
以下の様になると思います。
```suggestion
  List<Student> allStudents();
```

Reply:
また、メソッドは動詞ベースで命名しましょう。
なので修正前の`getAllStudents`の方が良いです